.travis.yml

```diff
@@ -30,6 +30,7 @@ before_script:
   - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
   - eval "$(/opt/chefdk/bin/chef shell-init bash)"
   - /opt/chefdk/embedded/bin/chef gem install coveralls # needed for chefspecs
+  - /opt/chefdk/embedded/bin/chef gem install fauxhai # needed for chefspecs
```

Comment:
This should already be installed via ChefDK. We use it all over the place without installing it.